.travis.yml

```diff
@@ -29,8 +29,7 @@ matrix:
     - php: hhvm-nightly
 
 before_script:
-  - composer self-update
-  - composer install --no-interaction --prefer-source --dev
+  - composer install --no-interaction --prefer-source
```

Comment:
It's faster :) composer has a local cache for all the files it downloads (you should also use `--prefer-dist`), and travis can cache that